uobtheatre/reports/schema.py

```diff
@@ -0,0 +1,55 @@
+import graphene
+from django.urls import reverse
+from graphene.types.datetime import DateTime
+from graphene.types.scalars import String
+
+import uobtheatre.reports.reports as reports
+from uobtheatre.utils.exceptions import GQLNonFieldException, SafeMutation
+from uobtheatre.utils.schema import AuthRequiredMixin
+
+available_reports = {
+    "PeriodTotals": {
+        "cls": reports.PeriodTotalsBreakdown,
+        "uri": "period_totals",
+    },
+    "OutstandingPayments": {
+        "cls": reports.OutstandingSocietyPayments,
+        "uri": "outstanding_society_payments",
```

Comment:
We should alow the api to change the base url of a repot (incase we save to s3 in the future). We could do this by adding "REPORT_PATH" to settings?

Reply:
These currently return paths (like /reports/outstanding_payments), instead of fully qualified. I think for now, best way is for the front end to say: if we have just a path, use the API as the base, else (if fully qualified url) we'll use that. Gives us scope to use s3 in future